code/datums/mood_events/generic_negative_events.dm

```diff
@@ -0,0 +1,101 @@
+/datum/mood_event/handcuffed
+	description = "<span class='warning'>I guess my antics have finally caught up with me..</span>\n"
+	mood_change = -1
+
+/datum/mood_event/broken_vow //Used for when mimes break their vow of silence
+  description = "<span class='boldwarning'>I have brought shame upon my name, and betrayed my fellow mimes by breaking our sacred vow...</span>\n"
+  mood_change = -8
+
+/datum/mood_event/on_fire
+	description = "<span class='boldwarning'>I'M ON FIRE!!!</span>\n"
+	mood_change = -8
+
+/datum/mood_event/suffocation
+	description = "<span class='boldwarning'>CAN'T... BREATHE...</span>\n"
+	mood_change = -6
+
+/datum/mood_event/burnt_thumb
+	description = "<span class='warning'>I shouldn't play with lighters...</span>\n"
+	mood_change = -1
+	timeout = 1200
+
+/datum/mood_event/cold
+	description = "<span class='warning'>It's way too cold in here.</span>\n"
+	mood_change = -2
+
+/datum/mood_event/hot
+	description = "<span class='warning'>It's getting hot in here.</span>\n"
+	mood_change = -2
+
+/datum/mood_event/creampie
+	description = "<span class='warning'>I've been creamed. Tastes like pie flavor.</span>\n"
+	mood_change = -2
+	timeout = 1800
+
+/datum/mood_event/slipped
+	description = "<span class='warning'>I slipped. I should be more careful next time...</span>\n"
+	mood_change = -3
+	timeout = 1800
+
+/datum/mood_event/eye_stab
+	description = "<span class='boldwarning'>I used to be an adventurer like you, until I took a screwdriver to the eye.</span>\n"
+	mood_change = -6
+	timeout = 2400
+
+/datum/mood_event/delam //SM delamination
+	description = "<span class='boldwarning'>Those God damn engineers can't do anything right...</span>\n"
+	mood_change = -6
+	timeout = 3000
+
+/datum/mood_event/shameful_suicide //suicide_acts that return SHAME, like sord
+  description = "<span class='boldwarning'>I can't even end it all!</span>\n"
+  mood_change = -10
+  timeout = 600
+
+/datum/mood_event/dismembered
+  description = "<span class='boldwarning'>AHH! I WAS USING THAT LIMB!</span>\n"
+  mood_change = -8
+  timeout = 3000
+
+/datum/mood_event/noshoes
+	 description = "<span class='warning'>I am a disgrace to comedy everywhere!</span>\n"
+	 mood_change = -3
+
+/datum/mood_event/tased
+	description = "<span class='warning'>There's no \"z\" in \"taser\". It's in the zap.</span>\n"
+	mood_change = -4
+	timeout = 1200
+
+/datum/mood_event/embedded
+	description = "<span class='boldwarning'>Pull it out!</span>\n"
+	mood_change = -6
+
+/datum/mood_event/table
+	description = "<span class='warning'>Someone threw me on a table!</span>\n"
+	mood_change = -2
+	timeout = 1200
```

Comment:
Does this stack or not? Because you can passivetable someone

Reply:
No moodlets stack, theyre unique.

Reply:
that is bad

Reply:
how is it? it prevents cheesing.

Reply:
oh i misread. i thought you meant "No, moodlets stack" as in yes they do stack